feed/tests/test_post_model.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+from feed.models import Post
+import django.contrib.auth
+
+
+User = django.contrib.auth.get_user_model()
+POST_TITLE = "Some First Post Title"
+POST_CONTENT = "Some first post content here"
+
+
+@pytest.fixture
+def users(db):
+    user0 = User.objects.create_user('user1', password='pass123')
+    user1 = User.objects.create_user('user2', password='pass321')
+    return [user0, user1]
+
+
+@pytest.fixture
+def post0(db, users):
+    return Post.posts.create(title=POST_TITLE, content=POST_CONTENT, author=users[0])
+
+
+@pytest.mark.django_db
+class TestPostModel:
+    # Testing different post attributes
+
+    def test_new_post(self, users, post0):
```

Comment:
The test checks user creation, 
just filter uses and assert for the created user against fixture data

same goes for post

Reply:
Done